### PR TITLE
Move jupyter notebook installation to envdlib

### PR DESCRIPTION
**Pull request by oubotong**
As suggested in the [feat: support serving #1228](https://github.com/tensorchord/envd/pull/1228)

- [x] `config.jupyter()` move to envdlib (Done)

To test this PR, simply do:
```python
envdlib = include("https://github.com/tensorchord/envdlib")
def build():
    base(os="ubuntu20.04", language="python3")
    envdlib.jupyter_notebook(8888, 8888) #Or any port mapping you want

### CODE REVIEW

**Comment by kemingy** [comment]
I think these two notebook functions can be put in the same file. You don't have to use different files for different functions.

**Comment by oubotong** [comment]
> I think these two notebook functions can be put in the same file. You don't have to use different files for different functions.

Yes, make sense. Let me fix it

File: src/notebook.envd [hunk 1]
```diff
@@ -24,3 +24,32 @@ def jupyter_lab(envd_port=8888, host_port=0, token="''"):
...
+def jupyter_notebook(envd_port=8888, host_port=8888, token="''"):
+    """Configure Jupyter Notebook.
+
+    Args:
+        envd_port (Optional[int]): port used by envd container (default=8888)
+        host_port (Optional[int]): port used by host, if not specified or equals to 0,
+            envd will randomly choose a free port, default is 8888
+        token (Optional[str]): access token
+    """
+    install.python_packages(["jinja2", "notebook"])
```
Comment (kemingy) [inline src/notebook.envd]: I wonder why `jinja2` is required

Comment (oubotong) [reply src/notebook.envd]: I think when I install jupyter notebook, this was related to an error message:
```
nbconvertconvert 7.2.5 has requirement jinja2>=3.0

Comment (oubotong) [reply src/notebook.envd]: Just tried removing `jinja2`, there is no more errors.

**Pull request closed without merging**